cache-refresh.R

```diff
@@ -4,19 +4,13 @@ library(lubridate)
 options(scipen=999)
 
 # set up remote connection to BETYdb
-# bety_src <- src_postgres(
-#   dbname = Sys.getenv('bety_dbname'),
-#   password = Sys.getenv('bety_password'),
-#   host = Sys.getenv('bety_host'),
-#   port = Sys.getenv('bety_port'),
-#   user = Sys.getenv('bety_user')
-# )
-bety_src <- src_postgres(dbname = "bety", 
-                         password = 'bety', 
-                         #host = 'terra-bety.default', 
-                         host = 'localhost', 
-                         user = 'bety', 
-                         port = 5432)
+bety_src <- src_postgres(
+  dbname = Sys.getenv('bety_dbname'),
```

Comment:
might want to use `ifelse(Sys.getenv('SHELL') == '', 'hello', Sys.getenv('SHELL'))` so you can have default values and it can work both with and without docker.